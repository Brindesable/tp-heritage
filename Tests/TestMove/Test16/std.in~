C circle1 0 0 10
S selection1 -20 -20 20 20
MOVE selection 30 30
LIST