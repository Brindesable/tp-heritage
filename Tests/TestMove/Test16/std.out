OK
#1 object(s) created
OK
#1 object(s) selected
OK
#1 object(s) moved
C circle1 30 30 10
